examples/role-cs/role-cs.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <LangVersion>11</LangVersion>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
```

Comment:
You probably want to bump this to `net8.0` as well. We saw [jobs fail in azure native due to the removal of 6.0 when runner images were updated](https://github.com/pulumi/pulumi-azure-native/pull/3868).

Reply:
I tried that first, but turns out there's a bigger issue with most of the native providers: https://github.com/pulumi/ci-mgmt/issues/1307
The installation is rather broken right now, before github upgraded the runners it installed `8.0` and then `3.1.301`, right now it's installing `8.0`, `6.0` then `3.1.301`.

Somehow `6.0` is on the path before `8.0` now and that does not support `LangVersion` 11.

Reply:
We'll first need to fix https://github.com/pulumi/ci-mgmt/issues/1307 before being able to upgrade the target framework